bin/-tarball-info.js

```diff
@@ -166,7 +166,7 @@ function generatePackageReference(version, tarballName) {
 function insertTarballsToPackageJson(fileLocation, options = {}) {
   const pkgInfo = require(fileLocation);
   if (options.isRelativeTarball) {
-    pkgInfo.version = `${pkgInfo.version}.${CurrentSha}`;
+    // pkgInfo.version = `${pkgInfo.version}`;
```

Comment:
This seems odd?

Reply:
checking but I think we were unnecessarily re-appending here